cni/network/network.go

```diff
@@ -1017,10 +1094,21 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 		numEndpointsToDelete = plugin.nm.GetNumEndpointsByContainerID(args.ContainerID)
 	}
 
+	endpointID := plugin.nm.GetEndpointID(args.ContainerID, args.IfName)
+
+	// Get secondary interface info from CNI state file and save them to ipamAddResult in CNI Delete call
+	secondaryInterfaceInfo, err := plugin.nm.GetNetworkSecondaryInterfaceInfo(endpointID)
```

Comment:
should not be specfic call for each interface, instead it should fetch all interface associated with that containerID

Reply:
will address in stateless cni